### ISSUE TITLE
[Doc] hunter_configuration_types link is archived

### ISSUE THREAD

**Issue by NeroBurner** [cpp-pm/hunter#303]
* I've read [Brief overview section](https://hunter.readthedocs.io/en/latest/overview.html) and do understand basic concepts. **[Yes|No]**
* I've read [F.A.Q. section](https://hunter.readthedocs.io/en/latest/faq.html) and there is no solution to my problem there. **[Yes|No]**
* I've read [Code of Conduct](https://github.com/cpp-pm/hunter/blob/master/.github/CODE_OF_CONDUCT.md), I promise to be polite and will do my best at being constructive. **[Yes|No]**
* I've read [Reporting bugs section](https://hunter.readthedocs.io/en/latest/contributing.html#reporting-bugs) carefully. **[Yes|No]**
* I've checked that all the `hunter_add_package`/`find_package` API used by me in the example is the same as in [documentation](https://hunter.readthedocs.io/en/latest/packages.html). **[Yes|No]**
* I'm using [latest](https://github.com/cpp-pm/hunter/releases) Hunter `URL`/`SHA1`. **[Yes|No]**

- doc page: https://hunter.readthedocs.io/en/latest/overview/customization/build-types.html
- doc source: https://github.com/cpp-pm/hunter/blob/master/docs/overview/customization/build-types.rst

The link points to https://github.com/ruslo/hunter/wiki/example.hunter_configuration_types which just leads to the archived project instead of the wiki

I'd like to know how I can tell hunter to build with the build type `RelWithDebInfo`, but unfortunately the documentation is dead

**Comment by NeroBurner** [comment]
found another occurence: https://hunter.readthedocs.io/en/latest/reference/user-variables.html#hunter-configuration-types

**Comment by rbsheth** [comment]
@NeroBurner does this help? https://github.com/cpp-pm/hunter/blob/master/docs/old-wiki/example.hunter_configuration_types.md

**Comment by NeroBurner** [comment]
yes, perfect! Should we forward the link to the mentioned old-wiki or migrate the old-wiki entry to an rst page?

**Comment by rbsheth** [comment]
Up to you - you can decide if you send over a PR 😄
